src/ext/java/org/opentripplanner/ext/flex/template/FlexAccessEgressCallbackAdapter.java

```diff
@@ -0,0 +1,39 @@
+package org.opentripplanner.ext.flex.template;
+
+import java.util.Collection;
+import org.opentripplanner.ext.flex.trip.FlexTrip;
+import org.opentripplanner.model.PathTransfer;
+import org.opentripplanner.street.model.vertex.TransitStopVertex;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.site.StopLocation;
+
+/**
+ * To perform access/egress/direct flex searches, this module (this package) needs these
+ * services. We do not want to inject the implementations here and create unnecessary
+ * hard dependencies. By doing this, we explicitly list all external services needed and make
+ * testing easier. This also serves as documentation.
+ * <p>
+ * The implementation of this interface will for the most part just delegate to the implementing
+ * OTP service - look in these services for the documentation.
+ */
+public interface FlexAccessEgressCallbackAdapter {
```

Comment:
I've often need something like this: some code needs only a tiny part of TransitService or TransitModel and I've approched this in various ways. I believe we should have a discussion about a unified way.

Reply:
There is not "one way" - it depends. But, it is not point of inventing something here, just follow the guideline og the Structural patterns listed here: https://en.wikipedia.org/wiki/Design_Patterns

Reply:
In this case I used the Adaptor.